transport/TransportImpl.cpp

```diff
@@ -69,40 +70,6 @@ class PacketReceiveJob : public jobmanager::Job
     void (TransportImpl::*_receiveMethod)(Endpoint&, const SocketAddress&, memory::UniquePacket packet, uint64_t);
 };
 
-class TcpConnected : public jobmanager::Job
-{
-public:
-    TcpConnected(TransportImpl& transport,
-        std::shared_ptr<Endpoint> endpoint,
-        const SocketAddress& source,
-        memory::UniquePacket packet)
-        : _transport(transport),
-          _endpoint(endpoint),
-          _source(source),
-          _packet(std::move(packet))
-    {
-    }
-
-    void run() override { _transport.internalIceTcpConnect(_endpoint, _source, std::move(_packet)); }
-
-private:
-    TransportImpl& _transport;
-    std::shared_ptr<Endpoint> _endpoint;
-    const SocketAddress _source;
-    memory::UniquePacket _packet;
-};
-
-class UnregisterEndpointsJob : public jobmanager::CountedJob
-{
-public:
-    UnregisterEndpointsJob(TransportImpl& transport) : CountedJob(transport.getJobCounter()), _transport(transport) {}
-
-    void run() override { _transport.internalUnregisterEndpoints(); }
-
-private:
-    TransportImpl& _transport;
-};
-
 class IceDisconnectJob : public jobmanager::Job
```

Comment:
@olofkallander, I noticed this is no longer used. Did we remove the code by mistake or this not make sense any longer and I can remove?